fyrox-impl/src/engine/hotreload.rs

```diff
@@ -89,10 +91,11 @@ impl SceneState {
                 // The entire node belongs to plugin, serialize it entirely.
                 // Take the node out of the graph first.
                 let (ticket, node) = scene.graph.take_reserve(handle);
-                let mut container = NodeContainer::new(node);
+                // Using Node::visit_as_option makes the guarantee that writing to the Node is safe (no uninitalized Node (Box dyn NodeTrait) ever appears)
```

Comment:
Again, do not mention types from different crate here. The rule here is that you can't mention types of the crates down the hierarchy from the crates up in the hierarchy.